0124-securing-iot-governance-through-staking-rewards.md

```diff
@@ -9,38 +9,63 @@
+Until then, we propose to reduce the opportunity cost of acquiring veIOT voting power by paying rewards to locked IOT positions whose holders have voted in the last 2 of 4 votes. IOT Working Group elections will count towards this, as will votes cast indirectly through a proxy.
 
-Once decentralized Oracles are implemented and pay IOT rewards to the veIOT holders who delegate to them, the cost of locking IOT will be offset by these delegation rewards, mitigating the current disincentive for participating in HIP votes.
+The availability of rewards is anticipated to encourage more individuals to lock more IOT for longer periods. As a result, more participants who are long-term aligned with the success of the IOT subnetwork will hold more veHNT, strengthening the network’s resistance to potential governance attacks.
 
-Until then, we propose to reduce the opportunity cost of acquiring veIOT voting power by paying rewards to locked IOT positions. This will encourage more individuals to participate in IOT governance using (larger) locked IOT positions, thereby strengthening the network's resilience against potential governance attacks during periods of low participation.
+This HIP requires that all future IOT votes must provide an ABSTAIN option, so that voters may fulfill their voting requirement under this HIP even if they do not wish to commit to a YES or NO vote.
+
+ABSTAIN votes are not counted in the tally, so a vote will pass with 66.7% of all votes other than ABSTAIN. However, ABSTAIN votes count towards quorum.
 
 ## Stakeholders
 
-All current and future holders of IOT and veIOT are stakeholders in this proposal. Participants who lock IOT benefit from rewards, and all IOT holders benefit from long-term health and stability of the network promoted by greater governance participation.
+All current and future holders of IOT and veIOT are stakeholders in this proposal. Participants who lock IOT benefit from rewards, and all IOT holders benefit from the long-term health and stability of the network promoted by greater governance participation.
 
 ## Detailed Explanation
 
 1. **Emissions Apportionment:**
-    - Allocate 4% of IOT emissions to lockup rewards for veIOT. This allocation will be sourced from the reward bucket assigned to Oracles in [HIP-52][hip-52], which is currently not distributed.
+    - Implement a sliding scale for emissions allocated to voting rewards for IOT based on the total amount of veIOT in existence in the current epoch:
+        - **0.5%** of emissions is assigned as a base level.
+        - **1%** of emissions if **total veIOT** is at least **35 billion**
+        - **2%** of emissions if **total veIOT** is at least **50 billion**.
+        - **3%** of emissions if **total veIOT** is at least **75 billion**
+        - **4%** of emissions if **total veIOT** is at least **100 billion**.
```

Comment:
```suggestion
        - **4%** of emissions if **total veIOT** is at least **100 billion**.
        - This may be implemented via a chain variable/config that is modified by the program multisig. Alerting should be established that allows the multisig to be aware of a need to modify this variable and it is expected that the change will be made within 4 epochs of passing a defined threshold. 
```

made up an SLA of "4" epochs. Can be higher but preferably not lower.